Sources/Clappr_tvOS/Classes/Extension/UIFocusEnvironment+ext.swift

```diff
@@ -0,0 +1,6 @@
+extension UIFocusEnvironment {
+    var isFocusable: Bool {
```

Comment:
Which is the testing strategy for this extension ?

Reply:
Changes made!
Commit: 6c47086069f015d36a39390ba72f863f8792b7f6